res/values/crdroid_strings.xml

```diff
@@ -99,9 +98,11 @@
     <string name="recents_omni_category">OmniSwitch</string>
 
 
-     <string name="device_lenovo_karate">Lenovo K6 Power</string>
-     <string name="device_lenovo_karate_summary">\nMaintainer - Ethansiva (sivanesan) </string>
+    <string name="device_lenovo_karate">Lenovo K6 Power (karate)</string>
```

Comment:
crdroid_strings.xml should not be modified 
res/values/crdroid_devices.xml should be modified for adding a new device 
res/xml/device_maintainers.xml to add your name as a maintainer to a device 

fix this please